tests/test_prepare_batch_hovernet

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) MONAI Consortium
```

Comment:
Why this file is not a python (.py) file?

Thanks.

Reply:
Thanks @Nic-Ma, I just made a mistake here, have fixed it.